com.unity.render-pipelines.high-definition/Documentation~/whats-new-13.md

```diff
@@ -4,6 +4,23 @@ This page contains an overview of new features, improvements, and issues resolve
 
 ## Added
 
+### Material Samples Transparency Scenes
+
+![](Images/HDRP-MaterialSample-ShadowsTransparency.png)
+
+These new scenes include examples and informations on how to setup properly transparents in your projects using different rendering methods (Rasterization, Ray Tracing, Path Tracing).
```

Comment:
Should I be reviewing these? It looks like they have been added to this PR but are for other features

Reply:
Probably bad merge on my end. Will try fix up later.

Reply:
I don't see them in the diff? Maybe you are diffing vs previous commits? (I merged master in the meantime, so new stuff appeared)